k8s-deploy/pkg/cli/job_cli.go

```diff
@@ -15,8 +15,16 @@
 package cli
 
 import (
+	"bytes"
+	"encoding/json"
 	"errors"
+	"fmt"
+	"io/ioutil"
+	"net/http"
 
+	"github.com/FederatedAI/KubeFATE/k8s-deploy/pkg/api"
+	"github.com/rs/zerolog/log"
+	"github.com/spf13/viper"
```

Comment:
I still don't think it is a good practice to move code pieces from `cli/api.go` into this file. Maybe we should further discuss the code arrangement / api format.